misc/pytorch_toolkit/text_recognition/text_recognition/utils/trainer.py

```diff
@@ -145,6 +145,7 @@ def __init__(self, work_dir, config):
             'CTCLossZeroInf', False)) if self.loss_type == 'CTC' else None
         self.out_size = len(self.vocab) + 1 if self.loss_type == 'CTC' else len(self.vocab)
         self.model = TextRecognitionModel(config.get('backbone_config'), self.out_size, config.get('head', {}))
+        print(self.model)
```

Comment:
is it needed?

Reply:
This is a good log message about a model.